src/framework/musesampler/internal/libhandler.h

```diff
@@ -189,7 +189,7 @@ struct MuseSamplerLibHandler
         destroy = (ms_MuseSampler_destroy)muse::getLibFunc(m_lib, "ms_MuseSampler_destroy");
 
         if (at_least_v_0_100) {
-            initSamplerInternal2 = (ms_MuseSampler_init_2)muse::getLibFunc(m_lib, "ms_MuseSampler_init_2");
+            initSamplerInternal2 = (ms_MuseSampler_init_2)muse::getLibFunc(m_lib, "ms_MuseSampler_init");
```

Comment:
I think a comment is needed here, why not init_2 (or the handler needs to be renamed)

Reply:
I'll drop this commit later (before merging the PR). This is a temporary workaround so users can also test the PR